addons/auth/.prettierrc.js

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+'use strict';
```

Comment:
did this change anything at all? curious since this a config file that exports an object and strict mode doesn't have any effect. Totally fine to keep it for consistency though.

Reply:
This was also one of the automatic updates when running `ember init` for the new blueprint changes. And I agree its good to keep for consistency.